src/main/java/it/gov/pagopa/payhub/activities/dto/ingestion/debtposition/InstallmentIngestionFlowFileDTO.java

```diff
@@ -0,0 +1,142 @@
+package it.gov.pagopa.payhub.activities.dto.ingestion.debtposition;
+
+import com.opencsv.bean.CsvBindAndJoinByName;
+import com.opencsv.bean.CsvBindByName;
+import com.opencsv.bean.CsvCustomBindByName;
+import com.opencsv.bean.CsvIgnore;
+import it.gov.pagopa.payhub.activities.util.csv.CsvOffsetDateTimeConverter;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.apache.commons.collections4.MultiValuedMap;
+
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class InstallmentIngestionFlowFileDTO {
+
+    @CsvIgnore
+    private Long ingestionFlowFileLineNumber;
+
+    @CsvBindByName(column = "azione", required = true)
+    private ActionEnum action;
```

Comment:
let's introduce italian fields just for `legacy` version
```suggestion
    @CsvBindByNames({
       @CsvBindByName(column = "azione", required = true, profile = "legacy"),
       @CsvBindByName(column = "action", required = true)
    })
    private ActionEnum action;
```

Reply:
I didn't use `@CsvBindByNames` because is deprecated, we just need the two `@CsvBindByName`